equipment_handler/source/sTransmitter.c

```diff
@@ -35,9 +34,17 @@
  *              Success of the function defined in sTransmitter.h
  */
 STX_return read_reg(uint8_t internal_address, uint8_t *answer) {
+    i2cSetBaudrate(SBAND_I2C, 400);
```

Comment:
We set the baudrate to 400 in lots of places in the code. We should just set it in halcogen

Reply:
I agree

Reply:
I've created a clickup task for this. I've leave this line here for now until someone modifies the obc_software halcogen file and removes any runtime speed changes from this code and others.